runtime/oti/j9protos.h

```diff
@@ -1258,7 +1258,7 @@ extern J9_CFUNC void  dbgDumpStack (J9VMThread *vmThread, IDATA delta);
 #ifndef _J9VMVISIBILITY_
 #define _J9VMVISIBILITY_
 extern J9_CFUNC IDATA  checkVisibility (J9VMThread* currentThread, J9Class* sourceClass, J9Class* destClass, UDATA modifiers, UDATA lookupOptions);
-extern J9_CFUNC IDATA  checkModuleAccess(J9VMThread *currentThread, J9JavaVM* vm, J9ROMClass* srcRomClass, J9Module* srcModule, J9ROMClass* destRomClass, J9Module* destModule, UDATA destPackageID, UDATA lookupOptions);
+J9_CFUNC IDATA  checkModuleAccess(J9VMThread *currentThread, J9JavaVM* vm, J9ROMClass* srcRomClass, J9Module* srcModule, J9ROMClass* destRomClass, J9Module* destModule, UDATA destPackageID, UDATA lookupOptions);
```

Comment:
I think this can be moved into `vm_internal.h` which is the module it is referred, and avoid including to other modules.